core/test/element_geometric_operations_test.cpp

```diff
@@ -87,3 +87,88 @@ TEST(ElementRotationTest, unit_out) {
   geometry::rotate_out_arg(u_out, a, dimless);
   EXPECT_EQ(a, u_out);
 }
+
+template <typename T> class ElementLessTest : public ::testing::Test {};
```

Comment:
Tests also in wrong file?

Reply:
moved to element_logical_operations_test.cpp